front/app/modules/commercial/insights/admin/components/TopBar/ProjectButton.tsx

```diff
@@ -30,30 +31,28 @@ const ProjectButtonContent = styled.span`
 `;
 
 const ProjectButton = ({ projectId }: ProjectButtonProps) => {
-  const locale = useLocale();
-
   const project = useProject({
     projectId,
   });
 
-  if (isNilOrError(project) || isNilOrError(locale)) {
+  if (isNilOrError(project)) {
     return null;
   }
 
   return (
     <div data-testid="insightsProjectButton">
-      <Button
-        locale={locale}
-        buttonStyle="secondary-outlined"
-        linkTo={`/projects/${project.attributes.slug}`}
-        fontSize={`${fontSizes.small}px`}
-        padding="4px 6px"
-      >
-        <ProjectButtonContent>
-          <T value={project.attributes.title_multiloc} />
-          <Icon name="link" className="linkIcon" />
-        </ProjectButtonContent>
-      </Button>
+      <Link to={`/projects/${project.attributes.slug}`} target="_blank">
```

Comment:
A `button` tag in a `a` tag is not super semantically correct, but the button has a openInNewTab prop for links, (yeah, weird name.), so we can easily prevent that :) You could also use the button props to add the icon and text, but no pressure there, the styling in there is not always what we want.

Reply:
Didn't know this prop existed but cool, definitely a better solution! :)